crypto/asn1/a_strex.c

```diff
@@ -280,6 +280,8 @@ static int do_dump(unsigned long lflags, char_io *io_ch, void *arg,
     t.type = str->type;
     t.value.ptr = (char *)str;
     der_len = i2d_ASN1_TYPE(&t, NULL);
+    if (der_len < 0)
```

Comment:
<= maybe?

Reply:
i thought that malloc(0) would return a pointer that can be passed to free() without causing an error, but yes <= is better.

Reply:
The result of `malloc(0)` can always be passed to `free`.  I agree `<=` is better here.